### PR TITLE
Add Sail the Blog! post.

### PR DESCRIPTION
**Pull request by rcaballeromx**
The post announces the new policy for blogs approved by the Steering Committee.

Signed-off-by: rcaballeromx <grca@google.com>

### CODE REVIEW

File: content/blog/2019/sail-the-blog/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+publishdate: 2019-01-31
+attribution: Rigs Caballero, Google
+weight: 75
+keywords: [community, blog, contribution, guide, guideline, event]
+---
+
+Welcome to the Istio blog!
+
+To make it easier to publish your content on our website, we
+[updated the content types guide](https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#choosing-a-page-type).
```
Comment (geeknoid) [inline content/blog/2019/sail-the-blog/index.md]: Link should just be (/about/contribute/...

Comment (rcaballeromx) [reply content/blog/2019/sail-the-blog/index.md]: @geeknoid Wouldn't that take readers to the old version of the page where the policy hasn't been updated?

Comment (geeknoid) [reply content/blog/2019/sail-the-blog/index.md]: When a link starts with /, then the link is relative to the current web site in which the specific page exists. So if they're on istio.io, then the link points to istio.io. If they're on preliminary.istio.io, then it points there.

When I merge the blog into the release-1.0 branch, then I can point it to preliminary.istio.io for that branch only.

Comment (rcaballeromx) [reply content/blog/2019/sail-the-blog/index.md]: Thanks for explaining. I'm modifying the link as you suggest.

**Pull request merged**